### ISSUE TITLE
add tags_include/tags_exact to be able to find elements of arbitrary classes such as XCUIElementTypeTextView

### ISSUE THREAD

**Issue by ncarroll-mdsol** [appium/ruby_lib#488]
iOS offers a UITextView element that can be used to display multiline text with special formatting. From their [docs](https://developer.apple.com/reference/uikit/uitextview):

> You typically use a text view to display multiple lines of text, such as when displaying the body of a large text document.

This element comes across in the source with the class `XCUIElementTypeTextView`.
I think it's a reasonable expectation that the `text()` method would match these elements. However, currently only elements with class `XCUIElementTypeStaticText` are matched: https://github.com/appium/ruby_lib/blob/master/lib/appium_lib/ios/element/text.rb#L5

Thoughts? I looked around the code and it doesn't seem too difficult. I may give it a shot! I think to do it properly, some of the generic methods / helpers should be modified to take an array of class names, rather than a single class.

**Comment by KazuCocoa** [comment]
Yes, currently `text()` only support `XCUIElementTypeStaticText` class since this methods had been only implemented against `UIAStaticText` before.

Do generic methods means like the following code?
(This example is only for XCUITest.)

```ruby
# @param [String] value
# @param [Array[String]] *args
def tags_include(value, *args)
    visible_elements = args.flat_map(&:tags)
    elements_include visible_elements, value
end

# @param [String] value
# @param [Array[String]] *args
def tags_exact(value, *args)
    visible_elements = args.flat_map(&:tags)
    elements_exact visible_elements, value
end
```

`tags()` comes from https://github.com/appium/ruby_lib/blob/master/lib/appium_lib/ios/helper.rb#L332 .

**Comment by KazuCocoa** [comment]
I'll implement like the above helper as v9.3.3

**Comment by KazuCocoa** [comment]
I added `tags_include/tags_exact` and change this issue's text

**Comment by ncarroll-mdsol** [comment]
Awesome, looks fantastic! Thank you :)

**Comment by KazuCocoa** [comment]
published ruby_lib 9.3.3
